finals-2024-01-16/code/5332-milana-sokolova/src/app/band-list/band-list.component.html

```diff
@@ -1 +1,59 @@
 <p>And the band list and everything else comes here</p>
+<div>
```

Comment:
Since only the bands component is mentioned in the routing, I will only be grading that functionallity.